vnext/Shared/JSI/RuntimeHolder.h

```diff
@@ -6,6 +6,8 @@
 namespace facebook {
 namespace jsi {
 
+enum RuntimeType { Chakra = 0, Hermes = 1, V8 = 2, V8NAPI = 3 };
```

Comment:
shouldn't this just use the same type as JSIEngine?

Reply:
JSIEngine is not defined in "Shared" layer .. but facebook::react::JSIEngineOverride is in "Shared" layer which we've now switched to ..